### DISCUSSION TITLE
Analytics

### DISCUSSION THREAD

**Discussion by reubenalfred** [Megabit/Blazorise#4836] [General]
Do Megabit have information on what Blazorise is being used for and the experience of the developers? If not then having this information, IMHO, would greatly aid in making strategic development decisions.

I mentioned about brand marketing in another post and no-one responded so not sure what Megabit thought about it. I think that some degree of regular brand awareness raising campaigns is important and so is SEO. SEO does work 😊

This is one way the community can help the project.

All we need to do is set goals, identify the tasks and the skills involved for each task, create documentation to achieve these tasks with links for further information and contact for mentoring.

All these activities will contribute to the growth of any software project.

Participation is one way we can ensure that the OSS product that we use stays alive.

**Comment by stsrki** [comment]
Based on our existing clients, there are no rules. They come from all kinds of industries. The only general similarity is that most of them are working on private, or in-house projects.

We did, in fact launch a survey last week, and we're still getting responses.

**Reply by reubenalfred** [reply]
That's great 👍 anything that calls for action is a good way to get individuals to participate in discussions I guess.
